### ISSUE TITLE
404 from 'device management page' link while syncing to central server 

### ISSUE THREAD

**Issue by radinamatic** [learningequality/ka-lite#4101]
**Branch**: develop  (continued from #4100) 

**Current Behavior**: While on Device management page (/management/zone/...), pressing the `Sync to central server!` button opens the  `Successfully launched data syncing job. After syncing completes, visit the device management page to view results.` info message.  

![404-device0](https://cloud.githubusercontent.com/assets/1457929/8688615/9442716c-2a9e-11e5-8555-88f4ea2b0740.png)  

A part from being redundant/self-referencing (user is already on the **Device management**), the link inside info message leads to 404 page:

![404-device](https://cloud.githubusercontent.com/assets/1457929/8688644/f3b749d8-2a9e-11e5-8343-63242eb9626e.png)  

**Expected Behavior**:  I'm not sure what is supposed to happen here, maybe simply:  

 `Successfully launched data syncing job. See **Sync Sessions** below for details.` 

**Steps to reproduce**:

1. Log in as Admin.
2. Go to Manage > Facilities tab.  
3. Select the device.
4. On **Device management** page press the `Sync to central server!` button.
5. Press the ` device management page` link inside the info message.

**Comment by aronasorman** [comment]
Working for me on the latest 0.14.x.

**Comment by radinamatic** [comment]
I'm really sorry to be the party-breaker, but no... :(

**Branch**: Windows installer @MCGallaspy complied yesterday (so technically 0.14.x) on Windows XP.

**Steps to reproduce**:  

1. Login as Admin
2. Go to `Manage > Facilities`
3. Press the `Sync Now!` button, and you will get the `Successfully launched data syncing job. After syncing completes, visit the device management page to view results.` message. 

![sync1](https://cloud.githubusercontent.com/assets/1457929/9141278/bc248c78-3d37-11e5-8045-3897beeafb66.png)  

4. Press the `device management page` link in the above alert message, to get to the sync results:

![sync2](https://cloud.githubusercontent.com/assets/1457929/9141338/3a6e75da-3d38-11e5-8e2c-daca03c02156.png)

Up to here, all is correct!  

5. But if you press the `Sync to central server button` again (from the above step):

![sync3](https://cloud.githubusercontent.com/assets/1457929/9141385/89f46d94-3d38-11e5-8783-a3e91e8d00ee.png)

6. Once you get the same alert message as before and you press the link `device management page` again, you'll get:

![sync4](https://cloud.githubusercontent.com/assets/1457929/9141460/26800cc2-3d39-11e5-9a25-93d6036202c1.png)

Basically, the link `device management page` inside the alert message **on the same device management page** seems incorrect and gives 404: 

`http://127.0.0.1:8008/management/zone//device//`

**Comment by radinamatic** [comment]
I can't reopen this issue, but it is still valid on my end.

**Comment by radinamatic** [comment]
On fresh Windows 8 install, this is what I get:

![fireshot capture - error i ka lite_ - http___127 0 0 1_8008_management_zon](https://cloud.githubusercontent.com/assets/1457929/9143611/b85a95b4-3d47-11e5-8225-30b5931d4a3f.png)  

Message `(ImportError: No module named yaml)` is a new one for me... :(

**Comment by aronasorman** [comment]
It's lacking the `compileymltojson` step. @MCGallaspy should be recreating a new installer now.

**Comment by radinamatic** [comment]
Retesting with the latest installer on Windows XP: no message `(ImportError: No module named yaml)` here, but I still get:

```
Page not found

Sorry, but the requested page could not be found.
```

with the steps from above... :(

**Comment by radinamatic** [comment]
Today, testing French interface on WinXP, still getting this:

![404-device-manage2](https://cloud.githubusercontent.com/assets/1457929/9184693/1a768ca0-3fb7-11e5-9531-9d3312047b73.png)

Plus some sync errors:

![404-device-manage](https://cloud.githubusercontent.com/assets/1457929/9184704/2998006a-3fb7-11e5-99d4-200fff05f580.png)

**Comment by rtibbles** [comment]
Fixed in #4456
